backend/application/resources/auth.py

```diff
@@ -0,0 +1,40 @@
+from ...domain.models.auth import check_pw_pattern
+from ...app import app
+from apiflask import APIFlask, APIBlueprint, abort
+import bcrypt
+from infrastructure.database import db
+from infrastructure.database.entities import User
+
+router = APIBlueprint("authentications", __name__)
+
+
+@router.post("/SignIn")
```

Comment:
Escreva o path das rotas em minúsculo. E "signin" é o equivalente a login. Cadastro chamamos de "signup".

```suggestion
@router.post("/signup")
```